Issue/publicize button layout

theck13 opened the pull request:
### Fix
Update ***Connect***/***Disconnect*** button layout in ***Publicize*** including margin, padding, and background.

![publicize_button](https://user-images.githubusercontent.com/3827611/40921100-24dbde6e-67d4-11e8-9a8b-3098a4227037.png)

### Test
1. Go to ***Sites*** tab.
2. Tap ***Sharing*** item under ***Configuration*** section.
3. Tap item under ***Connections*** section.
4. Notice ***Connect*** button.
5. Long-press ***Connect*** button.
6. Notice ***Connect*** button background.
7. Enter social media account credentials.
8. Notice ***Connect Another Account*** button.
9. Long-press ***Connect Another Account*** button.
10. Notice ***Connect Another Account*** button background.
11. Notice ***Disconnect*** button.
12. Long-press ***Disconnect*** button.
13. Notice ***Disconnect*** button background.

SylvesterWilmott commented:
The entire width of the bottom section is tappable here so lets extend the 'ripple' to 100% of that container.

Re: the disconnect button, is the height for that 36dp?

Are we also able to fix the shadow clipping in this view as well?

theck13 commented:
> The entire width of the bottom section is tappable here so lets extend the 'ripple' to 100% of that container.

I updated the view width in f911b9d.

> Re: the disconnect button, is the height for that 36dp?

The button has `12dp` top padding, `12dp` bottom padding, and `14sp` text size so it should be `38dp` total.

> Are we also able to fix the shadow clipping in this view as well?

I updated the layout in 43d6dfd.
